rastervision_gdal_vsi/requirements.txt

```diff
@@ -1,2 +1,2 @@
 rastervision_pipeline==0.21
-gdal==3.6.3
+gdal>=3.4.1
```

Comment:
I would additionally bound it from above to <=3.6.3 just to be safe.

Reply:
:white_check_mark: Will add to subsequent push to this branch.